apps/backend/src/datasources/postgres/EventLogsDataSource.ts

```diff
@@ -23,15 +23,25 @@ export default class PostgresEventLogsDataSource
     eventType: string,
     rowData: string,
     changedObjectId: string,
-    description?: string
+    description?: string,
+    impersonatingUserId?: number
   ) {
+    const isImpersonating =
+      impersonatingUserId != null && impersonatingUserId !== changedBy;
+
+    const actingUserId = impersonatingUserId ?? changedBy;
+
+    const updatedDescription = isImpersonating
```

Comment:
suggestion: Do you think we could maybe add `userId` into the description, that would make it more readable. so `(userId: 2 impersonating userId:1)`
![image](https://github.com/user-attachments/assets/93755761-3d7a-4946-b09f-ded6adf9e845)